apps/finance/contracts/Finance.sol

```diff
@@ -319,17 +319,17 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
     }
 
     /**
-    * @notice `_disabled ? 'Disable' : 'Enable'` payment `_paymentId`
+    * @notice `_active ? 'Active' : 'Inactive'` payment `_paymentId`
     * @param _paymentId Identifier for payment
-    * @param _disabled Whether it will be disabled or enabled
+    * @param _active Whether it will be active or inactive
     */
-    function setPaymentDisabled(uint256 _paymentId, bool _disabled)
+    function setPaymentStatus(uint256 _paymentId, bool _active)
         external
         authP(DISABLE_PAYMENTS_ROLE, arr(_paymentId))
```

Comment:
We should also rename this role, so it's less confusing. It can actually enable or disable existing payments, so I'd rename it to `MANAGE_PAYMENTS_ROLE`.